### ISSUE TITLE
Add a cli argument to completely disable . semgrepignore

### ISSUE THREAD

**Issue by CaledoniaProject** [semgrep/semgrep#9376]
Hi there,

I found `. semgrepignore` to be quite annoying, sometimes I need to audit both the source code and php vendor code, and semgrep ignores them by default.

I tried to use `semgrep ci --help | grep semgrepignore` and couldn't find a way around. I had to do `echo > . semgrepignore` to disable this feature.

Can you add an cli argument to do that?

**Comment by aryx** [comment]
I agree, I think it would be useful, especially because when there's no .semgrepignore we apply a default one

**Comment by brandonspark** [comment]
Seems that this exists now: https://github.com/semgrep/semgrep/commit/26c0b1b64e5d46e57b9b567358d412f58417a093
